stl/inc/type_traits

```diff
@@ -1804,6 +1804,40 @@ inline constexpr bool is_nothrow_invocable_r_v =
     _Select_invoke_traits<_Callable, _Args...>::template _Is_nothrow_invocable_r<_Rx>::value;
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+#ifndef __EDG__ // TRANSITION, VSO-1268984
+#ifndef __clang__ // TRANSITION, LLVM-Bug #: To Be Filed

```

Comment:
I could live with merging this as-is and fixing later, but it would be nice to have the LLVM bug# now. I or Stephan can post the report for you if the delay is getting an account up at bugs.llvm.org.

Reply:
Yes, I'm still waiting for the account approval. I'd like to go through the process of creating the LLVM bug, but would also be fine if you want to post the report.

Reply:
If you are interested in the full process then I won't stand in your way ;)